images/dockerregistry/config.yml

```diff
@@ -14,9 +14,14 @@ auth:
   openshift:
     realm: openshift
 middleware:
+  registry:
+    - name: openshift
   repository:
     - name: openshift
       options:
+        acceptschema2: false
         pullthrough: true
         enforcequota: false
         projectcachettl: 1m
+  storage:
+    - name: openshift
```

Comment:
Let's not forget to update [openshift-docs](https://github.com/openshift/openshift-docs/blob/master/install_config/install/docker_registry.adoc#deploying-updated-configuration).